Resources/Prototypes/Entities/Clothing/Shoes/misc.yml

```diff
@@ -39,20 +39,25 @@
   components:
   - type: Sprite
     sprite: Clothing/Shoes/Misc/duck-slippers.rsi
-  - type: Clothing
+  - type: ToggleClothing
```

Comment:
be sure to comment and document this change as a Frontier change to an upstream file. A simple `# Frontier` with an optional description of the change goes a long way in keeping the changes maintainable.